Scholars Portal Journals.js

```diff
@@ -0,0 +1,172 @@
+{
+	"translatorID": "5ac0fd37-5578-4f82-8340-0e135b6336ee",
+	"label": "Scholars Portal Journals",
+	"creator": "Bartek Kawula",
+	"target": "https?://journals[1-2]\\.scholarsportal\\.info/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-02-19 04:18:37"
+}
+
+function detectWeb(doc, url) { 
+	// see if saved list is toggled open
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+		return 'multiple'
+	} else {
+		if(url.indexOf("/search?q") != -1 || url.indexOf("/browse/") != -1){
+			return 'multiple'
+		}
+		else {
+			return 'journalArticle';
```

Comment:
that's not strict enough. It would display Zotero icon on the landing page. You need to perform some checks to be 99.9% sure that there is something to be scraped from the page.